packages/expo-mail-composer/ios/MailComposerModule.swift

```diff
@@ -37,9 +38,25 @@ public class MailComposerModule: Module {
       }
     }
     .runOnQueue(.main)
+
+    // Function to check the availability of listed mail clients on the device
+    Function("getClients") { () -> [MailClient] in
+      return self.getAvailableMailClients()
+    }
+  }
+
+  // Private function to check each mail client for availability
+  private func getAvailableMailClients() -> [MailClient] {
```

Comment:
This function doesn't seem to depend on `self`, so it might be better to move it outside the class.